### ISSUE TITLE
Chesapeake Dataset Subsets not returning images only mask

### ISSUE THREAD

**Issue by nilsleh** [microsoft/torchgeo#358]
I am trying to use just a state subset of the Chesapeake dataset with the following code:
```
ds = ChesapeakeDE("./data", download=True)
sampler = RandomGeoSampler(ds, size=100, length=10)
dl = DataLoader(ds, sampler=sampler, collate_fn=stack_samples)

for sample in dl:
    img = sample["image"] # empty list
    mask = sample["mask"] # tensor
```
However, this sample just an empty list for image, while the mask returns a tensor. Or is this the intended behavior, as the ```test_chesapeake.py``` file does not test for the sample containing the key ```"image"```? What would be the correct method to receive the image from the dataloader then?

Looking in the source code, there is a variable ```is_image``` in the Chesapeake Dataset Class that is set to False. When changing this variable to True, the sample returns a tensor for the key image, but then mask is an empty list, so it seems one can only have one or the other in a single DataLoader.

**Comment by adamjstewart** [comment]
The `ChesapeakeDE` dataset consists of semantic segmentation labels for the state of Delaware, it does not consist of any images. What are you trying to do? If you want to train a model to make these kinds of predictions, I would suggest pairing it with an imagery dataset like `NAIP` which has the same resolution.

**Comment by nilsleh** [comment]
Oh okay, I see. From the documentation it wasn't clear to me that this is only masks without imagery.

**Comment by adamjstewart** [comment]
This is true for almost all of the GeoDatasets, they are either image-only or mask-only. Feel free to submit a PR to clarify this in the docs.
